Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM photon:2.0 as builder
-MAINTAINER Venil Noronha <veniln@vmware.com>
```

Comment:
nit: let's keep the `MAINTAINER` line.

Reply:
Done. But why do we need `MAINTAINER` line?

Reply:
It's just a standard for Docker.